src/spectrecoff/Panel.fs

```diff
@@ -14,8 +14,8 @@ let defaultPanelLayout: PanelLayout =
        Sizing = Collapse
        Padding = AllEqual 2 }
 
-let customPanel (layout: PanelLayout) (header: string) (content: OutputPayload) =
-    let panel = Panel(content |> toMarkedUpString)
+let customPanel (layout: PanelLayout) (header: string) (content: string) =
+    let panel = Panel(content)
```

Comment:
No parantheses all the way I say!

```suggestion
    let panel = Panel content
```

Same in line 19 and 26

Reply:
Oh yeah. Aaaaaaall the way!